resilience4j-framework-common/src/main/java/io/github/resilience4j/common/circuitbreaker/configuration/CommonCircuitBreakerConfigurationProperties.java

```diff
@@ -707,6 +711,10 @@ public Duration getExponentialMaxWaitDurationInOpenState() {
 
         public InstanceProperties setExponentialMaxWaitDurationInOpenState(
             Duration exponentialMaxWaitDurationInOpenState) {
+            if (exponentialMaxWaitDurationInOpenState.toMillis() < 1) {
```

Comment:
Not sure why some values (like `maxWaitDuration`) are allowed to have below millisecond values and some not (like this one). I didn't changed it. Its validated during circuit breaker work done [here](https://github.com/resilience4j/resilience4j/blob/b81dd07aaa10a8f69fa983ed03e14f062048c552/resilience4j-core/src/main/java/io/github/resilience4j/core/IntervalFunction.java#L246)